Resources/Audio/Misc/attributions.yml

```
@@ -47,3 +47,8 @@
   license: "CC-BY-SA-3.0"
   copyright: "Taken from Citadel Station 13"
   source: "https://github.com/Skyrat-SS13/Skyrat13/commit/2d4f2d1b489590b559e4073f41b126cef56f4c50"
+
+- files: ["yellow.ogg"]
+  license: "CC0"
```

```suggestion
  license: "CC0-1.0"
```
Needs to have the correct name so that the linter does not complain.
However, I was unable to find a licence on the website you linked. Could you tell me where it says it is public domain?

---

It would still need a proper licence to be usable for us.
Taking a recording from a TV show is a big no.
Not sure if a direct, self-made recording of one of the alarms would work because it technically still belongs to the company creating them I guess? I'm no lawyer though.
In addition I personally prefer the old sound.